articles/appliance/geo-ha2.md

```diff
@@ -0,0 +1,48 @@
+# Auth0 Appliance: High Availability Geo Cluster (Geo HA)
+
+The high availability geo cluster is an Appliance implementation that provides data center redundancy and automatic failure handling. This is the highest form of Appliance availability offered by Auth0.
+
+## Overview
+
+Auth0 adds to the single data center high availability solution by extending the cluster with a geographically distributed data center with a **maximum latency between the two data centers of no more than 100ms (round trip)**. The result is the high availability geo cluster, which is an active hot standby configuration with automated failure handling that can survive a regional outage.
```

Comment:
Katie, I'd soften this a bit. I think we'd be willing to discuss a latency > 100ms if the load is pretty low, or if the customer can tolerate more lost data. Could this read something like `the maximum round trip latency should be no more than 100 ms`?